LanguageExt.Core/ClassInstances/Monad/MTaskFirst.cs

```diff
@@ -149,10 +149,7 @@ public struct MTaskFirst<A> :
         public async Task<A> Apply(Func<A, A, A> f, Task<A> fa, Task<A> fb) 
         {
             await Task.WhenAll(fa, fb);
-            return fa.IsFaulted && fb.IsFaulted ? throw new AggregateException(fa.Exception, fb.Exception)
-                 : fa.IsFaulted                 ? throw new InnerException(fa.Exception)
-                 : fb.IsFaulted                 ? throw new InnerException(fb.Exception)
-                 : f(fa.Result, fb.Result);
+            f(fa.Result, fb.Result);
```

Comment:
Missing `return`. It's probably worth making sure your code compiles before committing ;)

Reply:
Oh, sorry *sheepish grin*